cmd/snapctl/task.go

```diff
@@ -477,6 +480,10 @@ func listTask(ctx *cli.Context) error {
 		"LAST FAILURE",
 	)
 	for _, task := range tasks.ScheduledTasks {
+		lastErrorMessage := task.LastFailureMessage
+		if !ctx.Bool("verbose") && len(lastErrorMessage)+153 > termWidth && termWidth > 164 {
```

Comment:
154 is the number of characters in the output before the error:
61b55838-0605-430c-a209-2f3328880141   	 Task-61b55838-0605-430c-a209-2f3328880141     	 Running       	 13    	 0     	 6     	 4:17PM 8-29-2016  **error is printed here**

164 is 154 + "LAST FAILURE". So that if the title bar wraps then the message will wrap too. If the title bar doesn't wrap, then the error will truncate.